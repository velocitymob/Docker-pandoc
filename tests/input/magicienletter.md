---
header-includes: |
    \usepackage{xltxtra}
    \usepackage{xelatexemoji}
---

***

# ✨📜 La newsletter du(de la) magicien(ne) de l'integ LinkyDep 📜✨

***

## Les news du 19/12/2018 au 11/01/2019
### ✨ Nouveautés
- blablabla  
- blablabla

### 🔧 Mises à jour
- blablabla  
- blablabla

### 🐛 Corrections
- blablabla  
- blablabla

***

## Les infos [Bonux](https://fr.wikipedia.org/wiki/Bonux)

### 🐧 La commande du jour

Pour utiliser Ansible, c'est comme ca :  
`ansible-playbook -i inventory <votre_inventaire> <nom_du_playbook>`

### 💭 Emoji Mood de l'équipe
> 🐳

### 🌐 Les beaux articles du web
- [It’s time! Migrating to Java 11](https://medium.com/criciumadev/its-time-migrating-to-java-11-5eb3868354f9)
- [Faille 0-day pour VirtualBox, exploitable sur tous les systèmes](https://www.nextinpact.com/brief/faille-0-day-pour-virtualbox--exploitable-sur-tous-les-systemes-6427.htm?skipua=1)

### 💬 La citation du jour
> "Le véritable travail, c'est de savoir attendre." - Jean Rostand  


***

<center>
Pour contacter l'intégration, c'est par ici :
[DSI-SOC-LYON-DEP-INTEG](mailto:DSI-SOC-LYON-DEP-INTEG)  
Le magicien de la semaine c'est [par là](http://confluence.linkydep.enedis.fr/pages/viewpage.action?pageId=8224882) ou sur notre Kanban
</center>

***

Test emojis :

    Mage : 🧙
    Female Mage : 🧙‍♀️
    Fortune cookie : 🥠
    Thermometer : 🌡️
