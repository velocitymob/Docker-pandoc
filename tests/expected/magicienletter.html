<hr />
<h1 id="la-newsletter-dude-la-magicienne-de-linteg-linkydep">✨📜 La newsletter du(de la) magicien(ne) de l’integ LinkyDep 📜✨</h1>
<hr />
<h2 id="les-news-du-19122018-au-11012019">Les news du 19/12/2018 au 11/01/2019</h2>
<h3 id="nouveautés">✨ Nouveautés</h3>
<ul>
<li>blablabla<br />
</li>
<li>blablabla</li>
</ul>
<h3 id="mises-à-jour">🔧 Mises à jour</h3>
<ul>
<li>blablabla<br />
</li>
<li>blablabla</li>
</ul>
<h3 id="corrections">🐛 Corrections</h3>
<ul>
<li>blablabla<br />
</li>
<li>blablabla</li>
</ul>
<hr />
<h2 id="les-infos-bonux">Les infos <a href="https://fr.wikipedia.org/wiki/Bonux">Bonux</a></h2>
<h3 id="la-commande-du-jour">🐧 La commande du jour</h3>
<p>Pour utiliser Ansible, c’est comme ca :<br />
<code>ansible-playbook -i inventory &lt;votre_inventaire&gt; &lt;nom_du_playbook&gt;</code></p>
<h3 id="emoji-mood-de-léquipe">💭 Emoji Mood de l’équipe</h3>
<blockquote>
<p>🐳</p>
</blockquote>
<h3 id="les-beaux-articles-du-web">🌐 Les beaux articles du web</h3>
<ul>
<li><a href="https://medium.com/criciumadev/its-time-migrating-to-java-11-5eb3868354f9">It’s time! Migrating to Java 11</a></li>
<li><a href="https://www.nextinpact.com/brief/faille-0-day-pour-virtualbox--exploitable-sur-tous-les-systemes-6427.htm?skipua=1">Faille 0-day pour VirtualBox, exploitable sur tous les systèmes</a></li>
</ul>
<h3 id="la-citation-du-jour">💬 La citation du jour</h3>
<blockquote>
<p>“Le véritable travail, c’est de savoir attendre.” - Jean Rostand</p>
</blockquote>
<hr />
<center>
Pour contacter l’intégration, c’est par ici : <a href="mailto:DSI-SOC-LYON-DEP-INTEG">DSI-SOC-LYON-DEP-INTEG</a><br />
Le magicien de la semaine c’est <a href="http://confluence.linkydep.enedis.fr/pages/viewpage.action?pageId=8224882">par là</a> ou sur notre Kanban
</center>
<hr />
<p>Test emojis :</p>
<pre><code>Mage : 🧙
Female Mage : 🧙‍♀️
Fortune cookie : 🥠
Thermometer : 🌡️</code></pre>
